lib/transaction.js

```diff
@@ -122,6 +125,33 @@ module.exports = function transaction() {
   });
 };
 
+transaction.start = function start(modelBase) {
+  if (!modelBase) {
+    return Promise.reject(new Error('objection.transaction.start: provide at least one Model class to bind to the transaction'))
+  }
+
+  if (!utils.isSubclassOf(modelBase, Model)) {
+    return Promise.reject(new Error('objection.transaction.start: all but the last argument should be Model subclasses'));
+  }
+
+  var knex = modelBase.knex();
+
+  return new Promise(function(resolve, reject) {
+    knex.transaction(function (trx) {
+        modelBase = modelBase.knex(trx);
```

Comment:
This sets the the transaction globally for the model. Consider the case that 100 requests are currently running on your node server that use the `modelBase` class. You set the knex connection of _all_ of them here. Very bad things will happen. This line is actually useless.